content/blog/deploying-simple-jitsi-meet-server.md

```diff
@@ -0,0 +1,315 @@
+  };
+
+  services.openssh.enable = true;
+  users.users.root.openssh.authorizedKeys.keys = [
+    # don’t forget to add your SSH public key(s) here!
+    "ssh-ed25519 AAAA…"
+  ];
+
+  # if your disk device is called differently, you need to change this
+  boot.loader.grub.device = "/dev/sda";
+  fileSystems."/" = {
+    device = "/dev/sda1";
+    fsType = "ext4";
+  };
+
+  system.stateVersion = "22.05"; # change this to the NixOS version you first installed the system with
+}
+```
+
+Now, you can just run `deploy` and everything should be set up automatically.
+After it worked successfully,
+you can remove the `sshOpts = [ "-4" ];` line from the flake,
+because it should now have IPv6 connectivity.
+
+## Getting basic Jitsi Meet service running
+
+What makes using Jitsi Meet with NixOS so nice
+is that there is a module for it.
+That means, you can enable a basic — but working — Jitsi Meet instance
+just by adding the following to your system configuration:
+
+```nix
+{
+  services.jitsi-meet = {
+    enable = true;
+    hostName = "meet.example.com"; # change this to your domain
+  };
+
+  services.jitsi-videobridge.openFirewall = true;
+
+  security.acme = {
+    acceptTerms = true; # ensure that you have read the subscriber agreement (https://letsencrypt.org/repository/)
+    defaults.email = "nospam@example.com"; # change this to your email address
+  };
+
+  networking.firewall.allowedTCPPorts = [ 80 443 ];
+}
+```
+
+After you deploy this configuration,
+you should already have a working Jitsi Meet service.
+
+## Advanced configuration
+
+### Customisation
+
+For customising the features and look of the Jitsi Meet web application,
+there are two options available.
+
+One is the interface configuration (`interface_config.js`),
+which is deprecated but still required to configure some things.
+It can be modified by setting the NixOS option `services.jitsi-meet.interfaceConfig`.
+A commented list of all options and their default values can be found [in the Jitsi Meet source code](https://github.com/jitsi/jitsi-meet/blob/master/interface_config.js).
+An example configuration that hides the Jitsi watermark could look like this:
+
+```nix
+{
+  services.jitsi-meet.interfaceConfig = {
+    SHOW_JITSI_WATERMARK = false;
+  };
+}
+```
+
+In the future, all options from `interface_config.js` will be moved to `config.js`.
+Many options are already configurable from it.
+Like with the interface configuration, a commented list of all options and their default values can be found [in the Jitsi Meet source code](https://github.com/jitsi/jitsi-meet/blob/master/config.js).
+There is one option I recommend you to set,
+`prejoinPageEnabled`, which enables a page to set up microphone and camera
+and lets the user disable them *before* joining a meeting.
+There are many more options available,
+so it might be useful to look at the options and set up everything to your liking.
+An example configuration could look like this:
```

Comment:
```suggestion

There are many more options available,
so it might be useful to look at the options and set up everything to your liking.

An example configuration could look like this:
```